src/v0/destinations/customerio/util.js

```diff
@@ -158,6 +159,9 @@ const identifyResponseBuilder = (userId, message) => {
     set(rawPayload, 'anonymous_id', message.anonymousId);
   }
   const endpoint = IDENTITY_ENDPOINT.replace(':id', userId);
+  if (validateEmail(userId)) {
```

Comment:
Function name should be isEmail, you are actually validating but just checking the format

Reply:
Done